### ISSUE TITLE
Are you interested in pushing for this feature to be built in to firefox?

### ISSUE THREAD

**Issue by PatrickGuthridge** [haxiomic/firefox-multi-touch-zoom#27]

**Comment by haxiomic** [comment]
Yeah, last I heard there were people working on it but I think progress it slow because of other bugs: https://bugzilla.mozilla.org/show_bug.cgi?id=688990#c50 (but there is activity)

I've been asked to PR this to Firefox but I'm not happy with this version going into the core product – there's too many small bugs which can't be fixed JS side

Work-wise I haven't got the free bandwidth to implement it natively anytime soon, so it's likely it'll arrive from the core team before I get round to doing it

**Comment by QWERTYUIOPYOZO** [comment]
It seems that every mention of it died about six months ago.

**Comment by haxiomic** [comment]
The latest activity is on this issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1503514
